examples/userapi/03_subdomains.ipynb

```diff
@@ -842,8 +842,21 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "Operator `Kernel` run in 10.68 s\n"
+      "Operator `Kernel` run in 15.77 s\n"
      ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "PerformanceSummary([(PerfKey(name='section0', rank=None),\n",
```

Comment:
why is this being printed now and not before?

Reply:
Gonna assume env var from previous person running it was different